patroni/postgresql/bootstrap.py

```diff
@@ -145,7 +150,7 @@ def _post_restore(self):
         if os.path.exists(trigger_file):
             os.unlink(trigger_file)
 
-    def _custom_bootstrap(self, config):
+    def _custom_bootstrap(self, config: Any) -> bool:
```

Comment:
```suggestion
    def _custom_bootstrap(self, config: Union[Any, Dict[str, Any]]) -> bool:
```

From what I can see `config` is mainly a `dict`, but `pyright` thinks it can receive `None` given how the method is called.